common/src/main/scala/explore/utils/package.scala

```diff
@@ -150,16 +155,18 @@ def clearInputIcon[EV[_], A](
     .map(_ => <.i(ExploreStyles.ClearableInputIcon, ^.onClick --> ev.set(view)(None)))
     .orUndefined
 
+// TODO Move these to lucuma-ui
 extension (toastRef: ToastRef)
-  def info(text: String) =
+  def show(text: String, severity: Message.Severity = Message.Severity.Info): Callback =
     toastRef.show(
       MessageItem(
         content = <.span(Icons.InfoLight.withSize(IconSize.LG), text),
+        severity = severity,
```

Comment:
I know this is old news now, but it looks to me as though all toasts using this method use the `InfoLight` icon regardless of severity?

Reply:
Yup, good catch. We should probable make the whole `content: VdomNode` a parameter.

Reply:
Are the `built in` prime icons so different from the FA ones that they are not usable for the toasts?

Reply:
You can set the icon in the `MessageItem`, but it only seems to work for `Messages`, not `Toasts`. I'll look deeper and see if there is a way to make it work. But, if that doesn't work, maybe rather than passing the whole `content` as a parameter, we could select an FA icon based on the severity.